### PR TITLE
Added SeisCon copy function

### PR DESCRIPTION
**Pull request by kerim371**
This function is required by JUDI's judiVector:
https://github.com/slimgroup/JUDI.jl/blob/16aa54abc9e1a18d8bd59636895afbd90718b0b7/src/TimeModeling/Types/judiVector.jl#L156

This is going to solve the issue: https://github.com/slimgroup/JUDI.jl/pull/181#issuecomment-1514766230

Nevertheless it is not possible to use this function as `copy(con)` because of the following warning:
```
WARNING: both SegyIO and Base export "copy"; uses of it in module Main must be qualified
```

It should be used as `SegyIO.copy(con)`.
Most likely JUDI's judiVector should be edited in appropriate way.

### CODE REVIEW

File: src/types/SeisCon.jl [hunk 1]
```diff
@@ -20,6 +20,13 @@ function getindex(con::SeisCon, a::Colon)
     read_con(con, 1:length(con))
 end
 
+"""
+    copy(s::SeisCon)
+
+Create a copy of SeisCon object.
+"""
+copy(s::SeisCon) = SeisCon(s.ns, s.dsf, s.blocks)
```
Comment (mloubout) [inline src/types/SeisCon.jl:28]: copy is a Base function which is why you cmget that Warning. You need to import it then extend it and not export it 

```
Import Base.copy

copy(.....
```

You also need to call copy on the blocks or the new object gonna refer to the same block unsafely

**Review by mloubout — CHANGES_REQUESTED** [review]

File: src/types/SeisCon.jl (hunk 1 above)
Comment (kerim371) [reply src/types/SeisCon.jl:28]: right, thank you!

**Comment by kerim371** [comment]
Seems to be fixed and working

**Comment by kerim371** [comment]
it seems even with this copy function added JUDI still doesn't want to work with SeisCon and filter:
`MethodError: no method matching filter!(::SeisCon, ::SeisCon, ::Float32; fmin=0.001f0, fmax=5.0f0)`

File: src/types/BlockScan.jl [hunk 2]
```diff
@@ -7,3 +9,9 @@ mutable struct BlockScan
     summary::Dict{String, Array{Int32, 1}}
 end
 
+"""
```
Comment (mloubout) [inline src/types/BlockScan.jl:12]: Remove docstring as there is a lready a complete one in Base

**Pull request merged**